android_jni/avifandroidjni/src/main/java/org/aomedia/avif/android/AvifDecoder.java

```diff
@@ -58,5 +58,21 @@ public static boolean isAvifImage(ByteBuffer buffer) {
    * @return true on success and false on failure. A few possible reasons for failure are: 1) Input
    *     was not valid AVIF. 2) Bitmap was not large enough to store the decoded image.
    */
-  public static native boolean decode(ByteBuffer encoded, int length, Bitmap bitmap);
+  public static boolean decode(ByteBuffer encoded, int length, Bitmap bitmap) {
+    return decode(encoded, length, bitmap, 0);
+  }
+
+  /**
+   * Decodes the AVIF image into the bitmap.
+   *
+   * @param encoded The encoded AVIF image. encoded.position() must be 0.
+   * @param length Length of the encoded buffer.
+   * @param bitmap The decoded pixels will be copied into the bitmap.
+   * @param threads Number of threads to be used for the AVIF decode. Zero means use number of CPU
```

Comment:
There can be confusion sometimes between thread count and 'creating additional' threads.

Reply:
added a note pointing to maxThreads variable in avif.h.

Reply:
thanks, lgtm